internal/pkg/cli/deploy/workload.go

```diff
@@ -561,17 +572,23 @@ func (d *workloadDeployer) runtimeConfig(in *StackRuntimeConfiguration) (*stack.
 	}
 	images := make(map[string]stack.ECRImage, len(in.ImageDigests))
 	for container, img := range in.ImageDigests {
+		imageTag := img.customOrGitTag()
+		if container != d.name {
```

Comment:
Can we add a comment here explaining _why_ the logic is different for sidecars? 

If we come back to read this code in the future, I'd be curious why this is necessary in the first place 👀